packages/lib/src/toolbar/controls/DomainWidget/DomainWidget.module.css

```diff
@@ -20,18 +20,21 @@
   opacity: 0.5; /* edit button is itself disabled, so don't fade it more */
 }
 
-.popup {
-  composes: popup from '../../utils.module.css';
-  z-index: 2; /* above overflow and selector menus */
+.popupOuter {
+  position: absolute;
+  z-index: 2; /* above other floating elements (overflow menu, selectors, etc.) */
+  bottom: 1px; /* guarantees overlap with toolbar so the popup doesn't close when the pointer moves into it */
+  padding-top: 6px; /* matches other floating elements */
+  transform: translateY(100%);
```

Comment:
We used to have two wrappers: `popup` and `popupInner`, but Floating UI no longer requires the extra level nesting; `InteractionHelp` and `OverflowMenu` only need the `popupInner` styles (the background color and box shadow).

So I rename `popupInner` to `popup`, and since the old `popup` class is now only used in `DomainWidget`, I move it here and rename it to `popupOuter`.